packages/typo3-version-handling/README.rst

```diff
@@ -0,0 +1,17 @@
+
+==========================================================
+Version handling for PHP based rendering of docs.typo3.org
+==========================================================
+
+This package contains a PHP enum that can be used to determine allowed TYPO3
+versions for documentation interlinking. It also allows a matching between
+relative versions, for example `stable` and absolute minor versions, for
```

Comment:
i had to read it three times to understand the sencence. I would avoid "relative" and "absolute", maybe better:

descriptive versions like stable

version numbers like 12.4